.github/workflows/build-test.yml

```
@@ -0,0 +1,28 @@
+    branches: [ "master" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x, 22.x]
+
+    steps:
+    - uses: actions/checkout@v4
```

yamllint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)